scripts/app.js

```diff
@@ -89,8 +93,55 @@ document.addEventListener("keydown", function(event) {
 
         saveContentToCache();
     }
+
+    if (event.ctrlKey && event.code === "Slash") {
+        event.preventDefault();
+        zoom(5);
+    }else if (event.ctrlKey && event.code === "Equal") {
+        event.preventDefault();
+        zoom(-5);
+    }else if (event.ctrlKey && event.code === "Digit0") {
+        event.preventDefault();
+        resetZoom();
+    }
+});
+
+// Zoom text
+buttonZoomPlus.addEventListener("click", function(event) {
+    event.preventDefault();
+    zoom(5);
+});
+
+buttonZoomMinus.addEventListener("click", function(event) {
+    event.preventDefault();
+    zoom(-5);
 });
 
+buttonZoomReset.addEventListener("click", function(event) {
+    event.preventDefault();
+    resetZoom();
+});
+
+function zoom(zoomValue) {
```

Comment:
To challenge a little:

- in the project, the font size is historically expressed in relative units (rem) and not in absolute units (px): it could be good to keep the notion of relative size
- on my screen, the base font size is 12.8px, which means that, if I understand the code correctly, zooming out is impossible and the maximum possible zoom is 37.8px (all possible values ​​in my case being: 12.8px, 17.8px, 22.8px, 27.8px, 32.8px and 37.8px)
The zoom feature as it stands remains interesting, but could be further improved in the future, if needed.

Reply:
Thank you for your feedback!

I’ve implemented a solution that works with rem instead of px to ensure the font size is relative. Here’s the approach I took:

```js
const fontSizeRem = parseFloat(window.getComputedStyle(input).fontSize) / parseFloat(window.getComputedStyle(document.documentElement).fontSize);
const lineHeightRem = parseFloat(window.getComputedStyle(input).lineHeight) / parseFloat(window.getComputedStyle(document.documentElement).fontSize);

const newFontSizeRem = fontSizeRem + zoomValue;
const newLineHeightRem = lineHeightRem + zoomValue;

if (newFontSizeRem < 0.5 || newFontSizeRem > 2) {
    return;
}

input.style.fontSize = `${newFontSizeRem}rem`;
output.style.fontSize = `${newFontSizeRem}rem`;
input.style.lineHeight = `${newLineHeightRem}rem`;
output.style.lineHeight = `${newLineHeightRem}rem`;
```

This approach should now allow zooming in and out while maintaining relative font sizing, as you suggested. The zoom range is currently set between 0.5rem and 2rem, but it can be adjusted if needed. Let me know what you think or if further adjustments are required!

Reply:
Looks good to me! 😄 Thank you!